llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -80,6 +80,12 @@ static Value getSlice(OpBuilder &b, Location loc, Value source,
       .Default([&](Type t) { return nullptr; });
 }
 
+// Returns true if two dimensions of a ShapedType are equal.
```

Comment:
if dimensions of ShapeType are dynamic or equal? (or better wording) but current comment seems misleading.
I would also give a more descriptive name to the function. (each shape has several dimensions)

Also extra nit, use `///` kind of comment for function documentation. (https://mlir.llvm.org/getting_started/DeveloperGuide/#style-guide)